src/one_dragon/base/screen/screen_utils.py

```
@@ -235,43 +270,50 @@ def is_target_screen(ctx: OneDragonContext, screen: MatLike,
 
-def find_by_ocr(ctx: OneDragonContext, screen: MatLike, target_cn: str,
-                area: Optional[ScreenArea] = None, lcs_percent: float = 0.5,
-                color_range: Optional[List] = None) -> bool:
+def find_by_ocr(
+    ctx: OneDragonContext,
+    screen: MatLike,
+    target_cn: str,
+    area: ScreenArea | None = None,
+    lcs_percent: float = 0.5,
+    color_range: list[list[int]] | None = None,
+    crop_first: bool = True,
```

Ruff (0.14.7)

Unused function argument: `crop_first`

(ARG001)